.github/workflows/linux.yml

```diff
@@ -52,7 +52,7 @@ jobs:
       with:
         platform: ubuntu22
         commit_packages_to_provide: wheels
-        revision: 747d0e7e105c9f2c9966a37861f95b1c7f886868
```

Comment:
please, update all other places where this commit is used 747d0e7e105c9f2c9966a37861f95b1c7f886868

It can be done once a PR to OpenVINO Tokenizers will be merged, so we can perform final validation